web/src/integTest/java/com/microsoft/applicationinsights/azuretest/IntegrationTestConfiguration.java

```diff
@@ -0,0 +1,131 @@
+package com.microsoft.applicationinsights.azuretest;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.io.File;
+
+/**
+ * Created by moralt on 30/4/2015.
+ */
+@XmlRootElement(name = "IntegrationTestConfiguration")
```

Comment:
Could have been much simpler but just using Java built in **[Properties](http://docs.oracle.com/javase/7/docs/api/java/util/Properties.html)** object